share/jupyter/voila/templates/classic/index.html.j2

```diff
@@ -36,7 +36,7 @@
     }
     </script>
     {# from this point on, nb.cells contains output of the executed cells #}
-    {% do notebook_execute(nb, kernel_id) %}
```

Comment:
Lets not change the API in this PR, also, who says you can only have 1 kernel? :)

Reply:
OK, it's just that it's not used at the moment.